src/main/kotlin/com/atlassian/performance/tools/report/api/result/StatsMeter.kt

```diff
@@ -10,6 +11,13 @@ import java.time.Duration
 
 class StatsMeter {
 
+    private companion object {
+        private val DEFAULT_TRIMMING: OutlierTrimming = OutlierTrimming(0.0, 1.0)
+    }
+
+    @Deprecated(
+        message = "Use overloaded measurePerformance instead. This method provides the same OutlierTrimming for every ActionMetric"
```

Comment:
> Use overloaded measurePerformance instead

Can't do it, it's `internal`.

> This method provides the same OutlierTrimming for every ActionMetric

Doesn't sound too bad.

Perhaps we could leave it un-deprecated?
Unless we want to deprecate it to keep API narrower and easier to maintain.